vnext/ReactUWP/Views/ScrollViewManager.cpp

```diff
@@ -206,32 +318,91 @@ void ScrollViewManager::UpdateProperties(ShadowNodeBase* nodeToUpdate, folly::dy
 
         scrollViewer.HorizontalScrollMode(horiz ? winrt::ScrollMode::Auto : winrt::ScrollMode::Disabled);
         scrollViewer.VerticalScrollMode(horiz ? winrt::ScrollMode::Disabled : winrt::ScrollMode::Auto);
+        ScrollViewUWPImplementation(scrollViewer).SetHorizontal(horiz);
```

Comment:
There is a stale PR in progress in this file, I think you're covering what they were trying to do with adding snap points.  but that change included some cleanup in this file that would be nice to have:  using propertyName/Value locals instead of pair.first/second, moving updateProperties to the shadownode.  and maybe wasn't in there but we should add handling for all these properties have a type of isNull() where it should go back to defaults when the property is unset.